vars/sectionNightlyTests.groovy

```diff
@@ -38,6 +38,15 @@ def call(PipelineCallbacksRunner pcr, AppPipelineConfig config, PipelineType pip
       }
     }
 
+    if (config.fortifyScan) {
+      fortifyScan(
+        pipelineCallbacksRunner: pcr,
+        fortifyVaultUrl: config.fortifyVaultUrl ?: "https://${product}-${environment.nonProdName}.vault.azure.net/",
```

Comment:
please use vault name not url,

We have no need for people to use something other than vault.azure.net domain and it makes the pipeline config a lot more verbose and is also different to db migration

https://github.com/hmcts/cnp-jenkins-library/blob/ccd-1257/src/uk/gov/hmcts/contino/AppPipelineConfig.groovy#L8

```suggestion
        fortifyVaultName: config.fortifyVaultName ?: "${product}-${environment.nonProdName}",
```

Reply:
done